modules/ROOT/pages/anypoint-monitoring-visualizer.adoc

```diff
@@ -0,0 +1,197 @@
+= Anypoint Monitoring and Visualizer
+
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Overall Architecture
```

Comment:
Let's move the whole architecture content to the runbook. This is public-facing customer docs and we don't want to mention how Monitoring and Visualizer are implemented or the underlying architecture. Only high-level product things about usage.

Reply:
Moved.